clients/android/NewsBlur/src/com/newsblur/activity/GlobalSharedStoriesItemsList.java

```diff
@@ -0,0 +1,93 @@
+package com.newsblur.activity;
+
+import android.app.FragmentTransaction;
+import android.content.ContentResolver;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuInflater;
+
+import com.newsblur.R;
+import com.newsblur.fragment.GlobalSharedStoriesItemListFragment;
+import com.newsblur.util.DefaultFeedView;
+import com.newsblur.util.FeedSet;
+import com.newsblur.util.PrefConstants;
+import com.newsblur.util.PrefsUtils;
+import com.newsblur.util.ReadFilter;
+import com.newsblur.util.StoryOrder;
+
+public class GlobalSharedStoriesItemsList extends ItemsList {
+
+	private ContentResolver resolver;
```

Comment:
Whoops, this member was cruft in the SavedStories activity.  No need for it, and I have cleaned it up from the original.